src/io.rs

```diff
@@ -39,6 +40,7 @@ impl Output {
     ///
     /// Sometimes alsa-lib writes to stderr, but if you prefer, you can write it here instead.
     /// Should you wish to empty the buffer; just call local_error_handler again and drop the old instance.
+    #[cfg(feature = "std")]
```

Comment:
Do you think we need to write in the comment to this function that it's unavailable in no-std mode or is this obvious when you see it in docs.rs later?